kubernetes/cray-opa/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-opa
-version: 1.13.0
+version: 1.13.1
```

Comment:
Can you bump this to 1.14.0? This is a big enough change that it shouldn't be a patch version.

However, due to how we do versioning you should not bump the minor number on the release/1.2 PR.